lib/core/services/jwt-helper.service.ts

```diff
@@ -36,6 +39,43 @@ export class JwtHelperService {
     constructor(private storageService: StorageService) {
     }
 
+    private globalRolesSubject = new BehaviorSubject([]);
+    private applicationRolesSubject = new BehaviorSubject([]);
+
+    globalRoles$: Observable<any> = this.globalRolesSubject.asObservable();
+    applicationRoles$: Observable<any> = this.applicationRolesSubject.asObservable();
```

Comment:
is it `any`? or its a known type like `readonly string[]`? Just asking ^^ im not sure how the roles are defined :)